javatests/arcs/core/host/LifecycleTest.kt

```diff
@@ -236,6 +236,8 @@ class LifecycleTest {
                 "onReady:1.1:[first]",
                 "data.onUpdate:2.2",
                 "onUpdate:2.2:[first]",
+                "data.onUpdate:2.2",
+                "onUpdate:2.2:[first]",
```

Comment:
This feels odd.. Could it have been a flake that made this necessary? What happens if you run this test a bunch?

Reply:
Yes it is flaky, it FAILED in 12 out of 100: sometimes there are 12 elements instead of 11 (11 looks like it is more common), sometimes they change order too. It is already marked as flaky in the BUILD rule (b/157513871). Let me know if you want me to file a more specific bug.